src/styles/blocks/card.scss

```diff
@@ -0,0 +1,71 @@
+.card {
+  border: 1px solid $color-gray;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__description {
+    color: $text-main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+  &__code {
+    color: $text-secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    color: $text-main-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    margin-top: 16px;
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
```

Comment:
The `margin` property is not consistent with the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). You should use shorthand properties whenever possible. Try to use the shorthand `margin` property here.